reports/config.ini

```diff
@@ -0,0 +1,22 @@
+[DEFAULT]
```

Comment:
Please remove this file and add a .gitignore entry so it does not get checked-in to the repo.